client/src/App.tsx

```diff
@@ -1,20 +1,37 @@
-import {observer} from "mobx-react-lite";
-import {Error404} from "pages/Error404/Error404";
-import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
+import { observer } from "mobx-react-lite";
+import { Error404 } from "pages/Error404/Error404";
```

Comment:
Export **Error404** `as default` firstly, and
```suggestion
const Error404 = React.lazy(() => import("./pages/Error404/Error404"))
```